scripts/publish-documentation.sh

```diff
@@ -21,7 +21,7 @@ git checkout $RELEASE_BRANCH
 git checkout $VERSION
 
 step "Installing dependencies…"
-./scripts/wcarthage.sh bootstrap --cache-builds
+carthage bootstrap --cache-builds --use-xcframeworks
```

Comment:
I do not see any `wcarthage.sh` mentioning in our docs and `publisher-production` branch, so I guess such change should be enough.